src/main/java/uk/ac/cam/cl/dtg/segue/api/UsersFacade.java

```diff
@@ -281,27 +276,28 @@ public Response createOrUpdateUserSettings(@Context final HttpServletRequest req
     }
 
     /**
-     * An endpoint to be refactored out of this class, which provides access to subject interests.
+     * Provides access to user preferences, mapping preference types to an inner map of preference names and values
      * @param httpServletRequest - the request, to work ou the current user
-     * @return subject interest map
+     * @return user preference map
      */
     @GET
-    @Path("users/subject_interests")
+    @Path("users/user_preferences")
     @Produces(MediaType.APPLICATION_JSON)
     @GZIP
-    public Response getSubjectInterests(@Context final HttpServletRequest httpServletRequest) {
-        // FIXME - this endpoint does not belong here, and should probably return actual user preferences eventually.
+    public Response getUserPreferences(@Context final HttpServletRequest httpServletRequest) {
```

Comment:
How often do we want to request all user preferences in one go as opposed to just those of a certain type? Do we want to allow for a filter param?